### ISSUE TITLE
examples for string properties with format `time` show "string"

### ISSUE THREAD

**Issue by c-pius** [asyncapi/asyncapi-react#209]
**Description**

Example values for `{type: "string", format: "time"}` objects return `"string"` instead of a time value.

**Expected result**

Examples for time formatted strings should be something like `"14:15:22"` 

**Actual result**

time property reads `"string"`

![image](https://user-images.githubusercontent.com/22994291/102018765-b3ac2680-3d6f-11eb-867b-ae40034ee3f4.png)

**Steps to reproduce**

Add the following to an arbitrary payload

```json
"some-time": {
    "type": "string",
    "format": "time"
}
```

**Troubleshooting**

I assume this is an inherited problem from the sample function of openapi-sampler
https://github.com/asyncapi/asyncapi-react/blob/36d3a21dabedb0e8c74876b7626f9a8045901ea3/library/src/helpers/generateExampleSchema.ts#L6

**Comment by magicmatatjahu** [comment]
@c-pius Hello! Yes, this is a problem with mentioned 3rd package. However, AsyncAPI supports this same formats like in OpenAPI https://github.com/asyncapi/asyncapi/blob/master/versions/2.0.0/asyncapi.md#dataTypeFormat Also in OpenAPI docs https://swagger.io/docs/specification/data-models/data-types/#string you can see that it supports `date` and `date-time` without mentioning `time`, so `openapi-sampler` package doesn't support it. `format` field is an open value, but probably tools go only with defined format in official docs. Only what we can do is make contribution to `openapi-sampler` package, and the bump it in our component, because issue is not related to component itself - making example for `time` inside component isn't an option.

**Comment by magicmatatjahu** [comment]
@c-pius I created issue for that https://github.com/Redocly/openapi-sampler/issues/120

**Comment by c-pius** [comment]
great, thanks for reaching out to those folks! Actually, I must admit that I wasn't aware that `time` format is not defined as part of OpenAPI 3

**Comment by derberg** [comment]
@c-pius yeah, OpenAPI 3 is actually not 100% JSON Schema. I think they want to fix it with OpenAPI 3.1 and `time` format came out in JSON Schema Draft 7 so "after" OpenAPI 3

**Comment by derberg** [comment]
@magicmatatjahu have a look at my comment, because of what I wrote, they might not agree to add support for `time`. Let's give Roman a few days to answer there, if he won't find the time, I'll try to catch him on one special Slack workspace 😄 

but we might end up that adding support for `time` might be rejected and we will have to think about alternative solutions, but yeah, let's wait for Roman

**Comment by magicmatatjahu** [comment]
@c-pius We merged `1.0.0` of `openapi-sampler` package to master (latest npm tag) and in `next` branch (@asyncapi/asyncapi-react@next package) :) If you will find some bugs, please reopen issue :)
